match_channels.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+import sys
+import pickle
+import numpy as np
+import torch
+
+
+def main():
+    file1in = open(sys.argv[1], "rb")
+    activs1 = pickle.load(file1in)
+    file1in.close()
```

Comment:
Kind of a nitpicky thing, but I would suggest using something like the following rather than manually closing the file
`
with open(sys.argv[1], "rb") as file1in:
    activs1 = pickle.load(file1in)
`